vocabularies/sample-type.ttl

```diff
@@ -43,6 +47,7 @@
         sampty:photograph,
         sampty:polished-block,
         sampty:powdered-chip,
+        sampty:pulp,

```

Comment:
I have added a comment below where the definition in. I have not heard the term pulp used in geology.

Reply:
commented below